WebViewControl/WebView.CefRequestHandler.cs

```diff
@@ -73,7 +73,8 @@ private class CefRequestHandler : RequestHandler {
                         exception = new RenderProcessCrashedException(ExceptionPrefix + "crashed");
                         break;
                     case CefTerminationStatus.ProcessWasKilled:
-                        exception = new RenderProcessKilledException(ExceptionPrefix + "was killed");
+                        var OnProcessKilledSufix = "The disposing status was: " + OwnerWebView.IsDisposing.ToString();
+                        exception = new RenderProcessKilledException(ExceptionPrefix + "was killed. " + OnProcessKilledSufix);

```

Comment:
You should add a property IsWebViewDisposing to the RenderProcessKilledException ... otherwise consumers won't be able to check if the webview is disposed. Surely they shouldn't be parsing the message string.